client/assets/js/search.js

```diff
@@ -15,22 +18,48 @@ async function getData(searchTerm) {
 }
 
 function updateUI(results) {
-    let resultsList = document.getElementById('resultsList')
+    let resultsList = document.getElementById('results');
+    let count = results.length;
+
+    let resultsCount = document.getElementById('results-count');
+    resultsCount.innerHTML = `<i>Number of search results: ${count}</i>`;
```

Comment:
Struggling to find things to comment on so maybe we can find an alternative to innerHTML so you don't get hacked potentially